test/helpers/kubectl.go

```diff
@@ -917,32 +916,22 @@ func (kub *Kubectl) CiliumReport(namespace string, commands ...string) {
 // `deadlocks` or `segmentation faults` messages. In case of any of these
 // messages, it'll mark the test as failed.
 func (kub *Kubectl) ValidateNoErrorsOnLogs(duration time.Duration) {
+	var logs string
 	cmd := fmt.Sprintf("%s -n %s logs --timestamps=true -l k8s-app=cilium --since=%vs",
 		KubectlCmd, KubeSystemNamespace, duration.Seconds())
 	res := kub.Exec(fmt.Sprintf("%s --previous", cmd), ExecOptions{SkipLog: true})
```

Comment:
Only check the previous logs for each individual pod if the restart count is nonzero for the given pod.

Reply:
The issue here is that we use `-l k8s-app=cilium` so we don't know the pod at all, if we want to add the pod it'll be more calls and I don't really want to do that, each call will be executed after each test, and that will take time and the time of the test will be increased.